nemo_skills/pipeline/generate.py

```diff
@@ -197,7 +211,11 @@ def generate(
         "Can provide an experiment name or an experiment object if running from code.",
     ),
     config_dir: str = typer.Option(None, help="Can customize where we search for cluster configs"),
-    log_dir: str = typer.Option(None, help="Can specify a custom location for slurm logs."),
+    log_dir: str = typer.Option(None, help="Can specify a custom location for slurm logs. "),
+    output_base: str = typer.Option(
```

Comment:
maybe output_prefix? or output_base_name?

Reply:
Let's add it for other generation types as well, so that it's supported in all cases